test/set/symmetric_difference.cpp

```diff
@@ -2,8 +2,6 @@
 // Distributed under the Boost Software License, Version 1.0.
 // (See accompanying file LICENSE.md or copy at http://boost.org/LICENSE_1_0.txt)
 
-#include <boost/hana/assert.hpp>
```

Comment:
It is already coming from:
test/set/symmetric_difference.cpp:10
test/_include/laws/base.hpp:25
test/_include/support/tracked.hpp:11

And that's why tests ares also passing.